implementations/Trential.json

```diff
@@ -0,0 +1,131 @@
+                "P-256"
+            ],
+            "supports": {
+                "vc": [
+                    "1.1",
+                    "2.0"
+                ]
+            }
+        },
+        {
+            "id": "did:key:z82Lm4idNbwpdMqeeR9Jt1Pti9W7xDeaETVNGRiLZ4a1XaBZALDh5qZ1hJvjTgwTvCfe4n3",
+            "endpoint": "https://api.trential.dev/indisi-vc/issuer/credentials/issue",
+            "tags": [
+                "ecdsa-rdfc-2019"
+            ],
+            "supportedEcdsaKeyTypes": [
+                "P-384"
+            ],
+            "supports": {
+                "vc": [
+                    "1.1",
+                    "2.0"
+                ]
+            }
+        },
+        {
+            "id": "did:key:zDnaecCAAyR7cqdb39JKrc71AV5wujsvQvA7J2LRctmbFUyz6",
+            "endpoint": "https://api.trential.dev/indisi-vc/issuer/credentials/issue",
+            "tags": [
+                "ecdsa-sd-2023"
+            ],
+            "supportedEcdsaKeyTypes": [
+                "P-256"
+            ],
+            "supports": {
+                "vc": [
+                    "1.1",
+                    "2.0"
+                ]
+            }
+        },
+        {
+            "id": "did:key:zUC7CXttUsbVYVwQCmuvaNePRj8kn8gY9EhfHrK9tJHssoL9PFqjHcStndAb1tHLgq5p563boFKGKCt4zHTtcCvdVogfb3HjL9oGAWjweioYC2m7svK6hHHh2Mj3mxw6bjQK9cR",
+            "endpoint": "https://api.trential.dev/indisi-vc/issuer/credentials/issue",
+            "tags": [
+                "bbs-2023"
+            ],
+            "supports": {
+                "vc": [
+                    "1.1",
+                    "2.0"
+                ]
+            }
+        },
+        {
+            "id": "did:key:z6Mksp1UFB9CFNSao7BC9gVkHoHidRK8NrZNNLqFDkp5xUXC",
+            "endpoint": "https://api.trential.dev/indisi-vc/issuer/credentials/issue",
+            "tags": [
+                "eddsa-rdfc-2022"
+            ],
+            "supports": {
+                "vc": [
+                    "1.1",
+                    "2.0"
+                ]
+            }
+        }
+    ],
+    "verifiers": [
+        {
+            "endpoint": "https://api.trential.dev/indisi-vc/verifier/credentials/verify",
+            "tags": [
+                "vc2.0",
+                "ecdsa-rdfc-2019",
+                "BitstringStatusList",
+                "ecdsa-rdfc-2019",
+                "ecdsa-sd-2023",
+                "bbs-2023",
+                "eddsa-rdfc-2022"
+            ],
+            "supportedEcdsaKeyTypes": [
+                "P-256",
+                "P-384"
+            ],
+            "supports": {
+                "vc": [
+                    "1.1",
+                    "2.0"
+                ]
+            }
+        }
+    ],
+    "vpVerifiers": [
+        {
+            "endpoint": "https://api.trential.dev/indisi-vc/verifier/presentations/verify",
+            "tags": [
+                "vc2.0"
+            ],
+            "options": {
```

Comment:
So the VC API no longer has official support for `checks` https://w3c-ccg.github.io/vc-api/#verify-presentation

If you want to leave this in that is fine, we understand if you need specific options passed to your suite that might have been deprecated, but this is just to let you know that checks is no longer in the VC API and endpoints should know what checks the verifier is expected to make.